spacy/tokens/doc.pyx

```diff
@@ -275,6 +275,8 @@ cdef class Doc:
             deps = [dep if dep is not None else MISSING_DEP_ for dep in deps]
         if deps and not heads:
             heads = [0] * len(deps)
+        if heads and not deps:
+            deps = ['dep'] * len(heads)
```

Comment:
Nitpicking: while `black` doesn't support `pyx` so isn't helpful here, we typically use double quotes for strings throughout the code base. Just for consistency :-)

Reply:
Ah, I knew that was the style, I'll have to keep an eye out. My instinct is always to use double quotes for human-facing strings and single quotes for internal-use strings, though I don't know where I got that from.